CONTRIBUTING.md

```diff
@@ -81,11 +90,14 @@ Pull requests should be submitted early and often! When opening a pull request,
 * **[REF]** for refactoring existing code    
 
 <br>
-
-If, when you submit, your pull request is not yet ready to be merged, please also include the **[WIP]** prefix. This tells the development team that your pull request is a "work-in-progress", and that you plan to continue working on it.
+Pull requests should be submitted early and often (please don't mix too many unrelated changes within one PR)!
+If your pull request is not yet ready to be merged, please also include the **[WIP]** prefix (you can remove it once your PR is ready to be merged). 
+This tells the development team that your pull request is a "work-in-progress", and that you plan to continue working on it.
 
 Review and discussion on new code can begin well before the work is complete, and the more discussion the better!
-In the worst case scenario, if the development team decides to pursue a different path than you've outlined, they'll close the pull request. That's really not so bad! :smile:
```

Comment:
I might retain this line, unless it's generally agreed that it's unwanted. I think there's some value in indicating that some changes may not be accepted, especially in the context of encouraging discussion and agreement prior to putting in a huge amount of effort on the code.

Reply:
I removed it, since  the idea that some (more or less abstract) team just closes the PR sounded to me (taking a perspective of person who is new to open source) rather harsh and discouraging ;-) but it's very subjective opinion...

what about "Review and discussion on new code can begin well before the work is complete, and the more discussion the better! The development team can prefer a different path than you've outlined, so it's better to discuss it and get approval at the early stage of your work."

Reply:
Sounds great, but I would use "The development team *may* ..." rather than "The development team *can*...".